.env.template

```
@@ -8,4 +8,7 @@
 # Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
 # See the documentation for all the connection string options: https://pris.ly/d/connection-strings
 
-DATABASE_URL="XXXXXXXXX"
\ No newline at end of file
+SUPABASE_URL=https://tkgihjsgtzjjrbtlxihm.supabase.co
+SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InRrZ2loanNndHpqanJidGx4aWhtIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NjI5Mzk2MjAsImV4cCI6MjA3ODUxNTYyMH0.Z7bXG1bXo1u5v3b8K0v5nU5Y1Q1Z4H3jF2K5z1Y5X9I
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SUPABASE_ANON_KEY key should go before the SUPABASE_URL key

(UnorderedKey)